src/bin/uutils.rs

```diff
@@ -9,74 +7,84 @@
  * file that was distributed with this source code.
  */
 
+// spell-checker:ignore (acronyms/names) Gehring
+// spell-checker:ignore (rustlang/crates) clippy concat rustlang termwidth textwrap
+// spell-checker:ignore (uutils) coreutils sigpipe uucore uumain uutils
+// spell-checker:ignore (shell) busybox symlinks
+
 include!(concat!(env!("OUT_DIR"), "/uutils_crates.rs"));
 
+extern crate lazy_static;
+extern crate textwrap;
+extern crate uucore;
+
+use lazy_static::lazy_static;
 use std::collections::hash_map::HashMap;
 use std::io::Write;
-use std::path::Path;
 
-extern crate uucore;
-
-static NAME: &str = "uutils";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
+lazy_static! {
```

Comment:
I don't think it really matters, but we could easily avoid using `lazy_static` as there are only two functions.

Reply:
Since `NAME` is a global, used by `usage()`, how would you rewrite it?